bedrock.js

```
@@ -104,16 +218,17 @@
   //
   //     var object = {};
   //     _.extend(object, Bedrock.Events);
-  //     object.on('expand', function(){ alert('expanded'); });
+  //     object.on('expand', function() { alert('expanded'); });
   //     object.trigger('expand');
   //
-  var Events = Bedrock.Events = {
+  var Events = {
 
     // Bind an event to a `callback` function. Passing `"all"` will bind
     // the callback to all events fired.
     on: function(name, callback, context) {
-      if (!eventsApi(this, 'on', name, [callback, context]) || !callback) return this;
-      if (this._events === undefined) this._events = {};
+      if (!eventsApi(this, 'on', name, [callback, context])) return this;
+      if (!callback) return this;
```

should this be doing `typeof callback !== 'function'` And instead of just returning false maybe this should throw a `TypeError` so people know when they screw up vs silently failing.